ui/src/tool-panel/location-track/dialog/location-track-edit-dialog.tsx

```diff
@@ -332,7 +332,9 @@ export const LocationTrackEditDialog: React.FC<LocationTrackDialogProps> = (
         });
```

Comment:
Tähän tiedostoon on näköjään tehty automaattiformatointimuutoksia, jotka ovat epähuomiossa päätyneet tähän pullariin. Sinänsä kai ok että menee eteenpäin tässäkin.